test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,96 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
+    def test_compare_model_stub_conv_static_fx(self):
+        r"""Compare the output of static quantized conv layer and its float shadow module"""
+
+        def calibrate(model, calib_data):
+            model.eval()
+            with torch.no_grad():
+                for inp in calib_data:
+                    model(*inp)
+
+        def compare_and_validate_results(float_model, q_model, module_swap_list, data):
+            ob_dict = compare_model_stub_fx(
+                float_model, q_model, module_swap_list, data
+            )
+            self.assertEqual(len(ob_dict), 1)
+            for k, v in ob_dict.items():
+                self.assertTrue(len(v["float"]) == len(v["quantized"]))
+                for i, val in enumerate(v["quantized"]):
+                    self.assertTrue(v["float"][i].shape == v["quantized"][i].shape)
+
+        qengine = torch.backends.quantized.engine
+        qconfig = get_default_qconfig(qengine)
+        qconfig_dict = {"": qconfig}
+
+        model_list = [ConvModel(), ConvBNReLU()]
+
+        for float_model in model_list:
+            float_model.eval()
+
+            prepared_model = prepare_fx(float_model, qconfig_dict)
+
+            backup_prepared_model = copy.deepcopy(prepared_model)
+
+            backup_prepared_model = remove_qconfig_observer_fx(backup_prepared_model)
```

Comment:
Would be good to move remove_qconfig_observer_fx() into compare_model_stub_fx() to clean up the float model and qmodel passed in, so that user don't need to do it by themselves.